Templates/GenericWebHook-CSharp/function.json

```diff
@@ -5,6 +5,7 @@
             "type": "httpTrigger",
             "direction": "in",
             "webHookType": "genericJson",
+            "authLevel": "function",
```

Comment:
This is essentially the same problem as Service Bus Topic name vs Queue name. I believe Alexey is adding support for that now. I'll pull that commit out and we can add the "exclusive" rule to the binding.json. We'll need to hold this until we fix the enum portal issues anyway.